### DISCUSSION TITLE
授权我的另外一个电脑使用 但是找不到授权码

### DISCUSSION THREAD

**Discussion by SpringTang2021** [getlantern/lantern-forum-cn#643] [Q&A]
我的邮箱是my.journey@live.cn
我有另外一台新电脑 想授权新电脑也使用网络，但是下面这个界面感觉应该是给接受授权码的电脑使用的，我的问题是，我登录的这个电脑上，哪里可以找到授权码？可以帮我查一下吗？感谢。

![image](https://github.com/getlantern/lantern-forum-cn/assets/92971466/bf0cbdf2-9d23-4f3b-bbf7-46b3c4a7f9bf)

**Comment by MaxMKMax** [comment]
@SpringTang2021 您好! 请问您是没收到授权码对吗? 一般授权码会发到您的邮箱. 如果没收到, 请用链接提交问题.

请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱. 
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
